### ISSUE TITLE
Review and update footprints, final round of general updates for 2023

### ISSUE THREAD

**Issue by danrademacher** [GreenInfo-Network/seattle-building-dashboard#85]
Splitting this out from #57 

This is the list of buildings that need review, as [also documented here](https://docs.google.com/spreadsheets/d/1HQeAVg7MXhHhrCs_xtIWyc8tWlpnHgq8/edit#gid=2123591046).

It is possible some of these exist in the 2015 Seattle Open Data, but I suspect mostly we need to digitize these, and then provide an updated set of the building outlines for Tom to populate in the application.

Dashboard link | Data Issue
-- | --
[43](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=43) | Property is multi-building Boeing campus - MR
[87](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=87) | Building Footprint is not accurate - appears school was newly rennovated/rebuilt MR
[172](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=172) | South Seattle College Campus is currently displaying one building and a dot to the southeast of the site - MR
[190](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=190) | Wrong buildings selected, property is large warehouse - MR
[212](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=212) | Building currently displayed is tiny shack on the campus of a large school. - MR
[215](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=215) | The historic Rainier Bottling plant and Malt House building are next to eachother in Georgetown. The Bottling plant is North and the Malt House Building is South. Right now, they are displayed in reverse. Additionally the Bottling plant is lumping in another unrelated building a block away. - MR
[225](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=225) | Current footprint is tiny building on periphery of giant school building - MR
[226](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=226) | School has 4 structures on the property. I believe main structure is the only building actually benchmarked as the other structures appear to be portable classrooms and may not be conditioned. MR
[266](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=266) | Building display is across the street from actual building. - MR
[19489](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=19489) | Partial building captured by shape. - MR
[21610](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=21610) | Property is 6 building apartment complex, only 2 are currently displayed - MR
[23050](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=23050) | This is a ~25 building apartment complex benchmarked as a single property, currently only 2 buildings are displaying - MR
[23639](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=23639) | Multibuilding Apartment complex currently only displaying 2 shapes - MR
[24030](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=24030) | Existing property displayed is tiny building on edge of lot, bulk of square footage is the main building on lot, currently not displaying. - MR
[24150](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=24150) | Multibuilding Apartment complex - MR
[24160](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=24160) | Multi-building apartment complex currently only displaying 2 shapes. - MR
[24161](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=24161) | Multi-building apartment complex currently only displaying 2 shapes. - MR
[24162](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=24162) | 3 building apartment complex only displaying 2 buildings. - MR
[24248](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=24248) | Wrong building footprint displayed - MR
[24494](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=24494) | Property is actually a multi-building apartment complex. Currently displaying a single tiny building. - MR
[24495](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=24495) | Property is multi-building apartment complext currently only displaying a single building. - MR
[24496](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=24496) | Property is a 4 building apartment complex currently only displaying a single building. - MR
[24645](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=24645) | Currently there are only 2 buildings displaying - according the KC assessor data, there are 7  buildings on the lot - MR
[24854](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=24854) | Property is large multibuilding apartment complex currently displaying 2 buildings. - MR
[24857](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=24857) | Property is currently displaying tiny shack on large lot with a shopping center. - MR
[25131](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=25131) | Wrong building footprints display - Condo complex is long narrow building on the left side of the lot; everything else on the lot are car ports - MR
[25299](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=25299) | Multibuilding Apartment complex - MR
[25435](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=25435) | Currently only displaying 1 building; there are 4 buildings on the lot and the benchmarking record appears to correspond to all 4 buildings - MR
[25652](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=25652) | Four building condo property only displaying 1 building -MR
[25761](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=25761) | Property is 23 building apartment complex - currently only one building is displayed. Note that some structures on the site are car ports. - MR
[26026](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=26026) | Data viz is displaying adjacent carport as a building - MR
[37248](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=37248) | The historic Rainier Bottling plant and Malt House building are next to eachother in Georgetown. The Bottling plant is North and the Malt House Building is South. Right now, they are displayed in reverse. - MR
[49699](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=49699) | Property is multi-building public storage facility that spans ~20 buildings - MR
[49736](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=49736) | Only parts of the building displaying - MR
[49911](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=49911) | It appears the footprint is duplicated/overlapping? - MR
[50415](https://greeninfo-network.github.io/seattle-building-dashboard/#seattle/2021?layer=total_ghg_emissions&sort=total_ghg_emissions&order=desc&lat=47.62313&lng=-122.34075&zoom=17&building=50415) | Property is 440k SF multi-building campus, not single small building - MR

**Comment by DariyaGIN** [comment]
I am doing a quick check to see how many of these actually need to be digitized / whether some of them are in the 2015 Seattle Open Data.

**Comment by joseph-stout** [comment]
Just to follow up here, Dariya found only one building footprint that needed digitizing after checking the missing IDs in the Seattle Open Data file, and digitized it herself. So I am of the understanding that this is resolved.

**Comment by DariyaGIN** [comment]
Correct.  This task is now completed - I have reviewed all the new building footprints submitted by Mike.  Almost all of them did not need to be digitized - I was able to grab the outlines from the 2015 Seattle Open Data shapefile, I digitized one of the footprints, and I ran the script that carries out the rest of the updates to the footprints, as requested by Mike.  The updated file is in the project folder in the P drive.
